exonum-java-binding/testkit/src/main/java/com/exonum/binding/testkit/TestKitExtension.java

```diff
@@ -29,22 +29,88 @@
 import org.junit.jupiter.api.extension.ParameterResolutionException;
 import org.junit.jupiter.api.extension.ParameterResolver;
 
+/**
+ * Extension that injects TestKit into service tests. Register this extension with TestKit builder
+ * and a TestKit will be injected as a parameter, instantiated from given builder.
+ *
+ * Allows injecting TestKit into test context:
+ *
+ * <pre><code>
+ * @RegisterExtension
+ * static TestKitExtension testKitExtension = new TestKitExtension(
+ *     TestKit.builder()
+ *         .withService(TestServiceModule.class));
+ *
+ * @BeforeEach
+ * void setUp(TestKit testKit) {
+ *   // Set up logic
+ * }
+ *
+ * @Test
+ * void test(TestKit testKit) {
+ *   // Test logic
+ * }
+ * </code></pre>
+ *
+ * instead of:
+ *
+ * <pre><code>
+ * static {
+ *   LibraryLoader.load();
+ * }
+ *
+ * @Test
+ * void test() {
+ *   try (TestKit testKit = TestKit.forService(TestServiceModule.class)) {
+ *     // Set up logic
```

Comment:
That is not an apples-to-apples comparison. If we include @BeforeEach in the first,
than there is absolutely no problem including @BeforeEach in the second example. With @AfterEach, necessarily, to destroy it :upside_down_face: